Resources/Prototypes/_Goobstation/Loadouts/Jobs/misc.yml

```
@@ -64,4 +66,29 @@
+
+- type: loadout
+  id: Mothroach
+  storage:
+    back:
+    - MobMothroach
+
+- type: loadout
+  id: Goldfish
+  storage:
+    back:
+    - MobCarpGoldfish
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)